feddlib/core/FE/Domain_def.hpp

```diff
@@ -397,6 +397,38 @@ void Domain<SC,LO,GO,NO>::setMesh(MeshUnstrPtr_Type meshUnstr){
     mesh_ = meshUnstr;
 }
 
+
+template <class SC, class LO, class GO, class NO>
+void Domain<SC, LO, GO, NO>::initDummyMesh(MapPtr_Type map)
+{
+    MeshUnstrPtr_Type outputMesh = Teuchos::rcp( new MeshUnstr_Type( comm_) );
+
+    outputMesh->dim_ = this->dim_ ;
+	outputMesh->FEType_ = this->FEType_ ;
+	//outputMesh->rankRange_ =  this->rankRange_;
+
+    //outputMesh->elementMap_ = this->elementMap_ ;
+	outputMesh->mapUnique_ = map;
+	outputMesh->mapRepeated_ = map;
+	//outputMesh->edgeMap_  = this->edgeMap_  ;
+
+	//outputMesh->elementsC_ = this->elementsC_;
+	//outputMesh->edgeElements_ = this->edgeElements_;
+	//outputMesh->surfaceTriangleElements_ = this->surfaceTriangleElements_;
+
+   	/*outputMesh->pointsRep_ =  this->pointsRep_  ; 
+    outputMesh->pointsUni_ = this->pointsUni_; 
+
+    outputMesh->bcFlagUni_ = this->bcFlagUni_ ; 
+	outputMesh->bcFlagRep_ = this->bcFlagRep_ ;
+
+
+	outputMesh->edgesElementOrder_ = this->edgesElementOrder_;
+	outputMesh->numElementsGlob_ = this->numElementsGlob_  ; */
+    mesh_ = outputMesh;
+
+}
+
```

Comment:
What is this function for? Comments on what it is needed for would also be good :)

Reply:
Explanation is in the _decl file most of the time :)

Reply:
Okay, sorry I did not check that. 
Maybe a few words as to why we need a dummy mesh for the Lagrange multiplier would be good.

Reply:
Done.